### PR TITLE
feat: 2642 document variable help

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: core/builtins/custom_tags.py [hunk 1]
```diff
@@ -596,6 +596,11 @@ def sentence_case(text):
     return capfirst(text).replace("_", " ")
 
 
+@register.filter()
+def format_heading(text):
+    return text.replace("_", " ").capitalize()
```
Comment (efahy) [inline core/builtins/custom_tags.py:601]: As discussed, remove `capitalize` and make the enums the correct case, thus allowing ECJU to be all uppercase, for example

File: letter_templates/views/create.py [hunk 2]
```diff
@@ -48,3 +49,18 @@ def post(request):
             return error_page(None, "; ".join(error_messages))
 
         return redirect("letter_templates:letter_templates")
+
+
+class VariableHelp(TemplateView):
+    @staticmethod
```
Comment (efahy) [inline letter_templates/views/create.py:55]: I think a docstring comment here would be good, explaining the general concept of the business configureable enums and text being dynamically turned into a rendered help page

Comment (BenAndrew-Kainos) [reply letter_templates/views/create.py:55]: Done

File: core/builtins/custom_tags.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply core/builtins/custom_tags.py]: done

**Pull request merged**